Database/Corrections/sodQuestFixes.lua

```diff
@@ -2441,6 +2441,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.objectivesText] = {"Kill Furlbogs until Crab Treats drop, use them to feed a Young Reef Crawler and receive the rune."},
             [questKeys.requiredSpell] = -416049,
         },
+        [90146] = {
+            [questKeys.name] = "Mangle",
+            [questKeys.startedBy] = {{3566,2960}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.DRUID,
+            [questKeys.objectivesText] = {"Kill Flatland Prowlers or Prairie Wolf Alphas until one drops the idol, equip it and do as it says."},
```

Comment:
Just to streamline with the wording of similar runes.

```suggestion
            [questKeys.objectivesText] = {"Kill Flatland Prowlers or Prairie Wolf Alphas until the Idol of Ursine Rage drops. Equip it and follow its guidance."},
```

Reply:
LGTM!